apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -159,17 +159,17 @@ const BookingPage = (props: BookingPageProps) => {
         ),
       };
     }
-    if (!props.booking || !props.booking.attendees.length) {
+    if (!booking || !booking.attendees.length) {
       return {};
     }
-    const primaryAttendee = props.booking.attendees[0];
+    const primaryAttendee = booking.attendees[0];
```

Comment:
This could also work as 
`const [primaryAttendee] = booking.attendees;`
NIT*

Reply:
Should I change it? I don't mind the [0] here